pkg/api/vlabs/validate.go

```diff
@@ -101,16 +101,27 @@ func (o *OrchestratorProfile) Validate(isUpdate bool) error {
 							return fmt.Errorf("enableAggregatedAPIs requires the enableRbac feature as a prerequisite")
 						}
 					}
+				}
 
-					if helpers.IsTrueBoolPointer(o.KubernetesConfig.EnableDataEncryptionAtRest) {
-						sv, _ := semver.NewVersion(o.OrchestratorVersion)
-						cons, _ := semver.NewConstraint("<" + "1.7.0")
-						if cons.Check(sv) {
-							return fmt.Errorf("enableDataEncryptionAtRest is only available in Kubernetes version %s or greater; unable to validate for Kubernetes version %s",
-								"1.7.0", o.OrchestratorVersion)
-						}
+				if helpers.IsTrueBoolPointer(o.KubernetesConfig.EnableDataEncryptionAtRest) {
+					sv, _ := semver.NewVersion(o.OrchestratorVersion)
+					cons, _ := semver.NewConstraint("<" + "1.7.0")
```

Comment:
1.10?

Reply:
This validation was previously added for `EnableDataEncryptionAtRest`, not `enableDataEncryptionAtRestWithExternalKms`. The validation for `enableDataEncryptionAtRestWithExternalKms ` is right below this block, which requires `1.10`.